ballerina/Module.md

```diff
@@ -34,29 +34,29 @@ Follow one of the ways below to add the JAR in the file:
 
 ### Client
 To access a database, you must first create an
-[oracledb:Client](https://docs.central.ballerina.io/ballerinax/oracledb/latest/clients/Client) object.
+[`oracledb:Client`](https://docs.central.ballerina.io/ballerinax/oracledb/latest/clients/Client) object.
 The samples for creating an OracleDB client can be found below.
 
 #### Creating a Client
-This sample shows the different ways of creating the `oracledb:Client`.
+This sample shows the different ways of creating an `oracledb:Client`.
 
 The client can be created with an empty constructor, and thereby, the client will be initialized with the default properties.
 
 ```ballerina
 oracledb:Client|sql:Error dbClient = new ();
 ```
 
-The `dbClient` receives the host, username, and password. Since the properties are passed in the same order as they are defined
-in the `oracledb:Client`, you can pass them without named params.
+The `oracledb:Client` receives the host, username, and password. Since the properties are passed in the same order as they are defined
+in the `oracledb:Client`, you can pass them without named parameters.
 
 ```ballerina
 oracledb:Client|sql:Error dbClient = new ("localhost", "adminUser", "adminPassword", 
                               "ORCLCDB.localdomain", 1521);
 ```
 
-The `dbClient` uses the named params to pass the attributes since it is skipping some params in the constructor.
+In the example below, the `oracledb:Client` uses named parameterss to pass the attributes since it is skipping some parameters in the constructor.
```

Comment:
```suggestion
In the example below, the `oracledb:Client` uses named parameters to pass the attributes since it is skipping some parameters in the constructor.
```

Reply:
Fixed in 75ce567